AvaliadorDisciplinas.FRONTEND/src/pages/Disciplina/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from "react";
+import styles from "./Disciplina.module.css";
+import UserImage from "../../assets/icons/user_anonimous.svg";
+import NavMenu from "../../components/NavMenu";
+import { useAuth } from "../../contexts/AuthContext";
+import DataList from "../../components/DataList";
+import { getAll } from "../../services/disciplinaService";
+import ButtonWithIcon from "../../components/ButtonWithIcon";
+import External from "../../assets/icons/external.svg";
+import Dropdown from "../../components/Dropdown";
+import { atributosDisciplina } from "../../services/DadosEstaticos";
+
+function DisciplinaIndex() {
+  const { user } = useAuth();
+  const [disc, setDisc] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [atributo, setAtributo] = useState();
+  const [text, setText] = useState("");
+
+  useEffect(() => {
+    async function fetchData() {
+      setLoading(true);
+      const data = await getAll(atributo, text);
+      setDisc(data);
+      setLoading(false);
+    }
+
+    fetchData();
+  }, [atributo, text]);
+
+  function renderItem(item) {
+    return (
+      <div className={styles.itemContainer}>
+        <div className={styles.itemNota}>{item.nota}/5</div>
+        <div className={styles.itemConteudo}>
+          <span>{item.nome}</span>
+          <span>Periodo: {item.periodo}</span>
+        </div>
+        <div className={styles.itemButton}>
+          <ButtonWithIcon
+            icon={External}
+            onClick={() => console.log(item.nome)}
+            transparent
```

Comment:
Aqui no navegador esse atributo deu erro no console, porque ele não identifica esse atributo como boolean, o erro é resolvido se colocar desta forma:
```suggestion
            transparent="true"
```

Reply:
Corrigido!